pkg/cloudprovider/providers/gce/gce_disks_test.go

```diff
@@ -723,9 +723,9 @@ func (manager *FakeServiceManager) CreateRegionalDiskOnCloudProvider(
 	tagsStr string,
 	diskType string,
 	zones sets.String) error {
-	manager.createDiskCalled = true
-	diskTypeURI := gceComputeAPIEndpointBeta + "projects/" + fmt.Sprintf(diskTypeURITemplateRegional, manager.gceProjectID, manager.gceRegion, diskType)
 
+	manager.createDiskCalled = true
+	diskTypeURI := gceComputeAPIEndpoint + "projects/" + fmt.Sprintf(diskTypeURITemplateRegional, manager.gceProjectID, manager.gceRegion, diskType)
 	switch t := manager.targetAPI; t {
```

Comment:
Now that all calls to the GCE PD API are GA, can we remove the checks for other versions?

Reply:
fixed

Reply:
Is it safe to remove the switch block entirely?